### ISSUE TITLE
deploy geo LSTB

### ISSUE THREAD

**Issue by teoteman** [jetkokos/tb_geo#2]
Could this code be used to develop a LS? is there a way to contribute under your domain to have both on it?

**Comment by promandev** [comment]
That's what I'm trying to do since yesterday. I've had a look and there is no README in this repo in order to give us some info (install stuff, how to run it...)

**Comment by Volet12** [comment]
Any update on LS GEO TB?

**Comment by teoteman** [comment]
No, owner will not do anything else, wouldve contacted by now. I'm doing a
slide for it

**Comment by genskaar** [comment]
I have forked the project and have created a LS GEO map which is being updated regularly. See https://github.com/genskaar/tb_geo . There are also some changes to the DS map squads and some style changes.
